wca-regulations.md

```diff
@@ -213,11 +213,13 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 9b1a) Round format for these events is: "Average of 5". For a Combined Round, the round format is "Best of 2" for the first phase and "Average of 5" for the second phase.
     - 9b2) 6x6x6 Cube and 7x7x7 Cube.
         - 9b2a) Round format for these events is: "Mean of 3". For a Combined Round, the round format is "Best of 1" for the first phase and "Mean of 3" for the second phase.
-    - 9b3) 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded, and 3x3x3 Multi-Blind.
+    - 9b3) 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded.
         - 9b3a) Round formats for these events are: "Best of X" (where X is 1, 2, or 3).
```

Comment:
There will probably be debate on whether or not a combined round should even be allowed for these events since it is easier to DNF (especially 4 and 5, not sure about 3) compared to multi.